packages/rocketchat-livechat/client/views/sideNav/livechat.js

```diff
@@ -36,6 +36,29 @@ Template.livechat.helpers({
 			}
 		});
 	},
+	inquiries() {
+		var user = Meteor.user();
+
+		// get all inquiries of the department 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726498730/issues/source?bid=3199634&fileBranchId=3348698#l42)